packages/@sanity/form-builder/src/inputs/BlockEditor/nodes/styles/ListItem.css

```diff
@@ -70,49 +70,42 @@
   composes: root;
 
   @nest & > .item:before {
-    content: '\25CF '; /* BLACK CIRCLE */
+    content: '  \25CF'; /* BLACK CIRCLE */
+    font-size: 0.4375rem; /* 7px */
+    line-height: 1.5rem; /* Same as body text */
+    /* Optical alignment */
+    position: relative;
+    top: 1px;
   }
 
-  @nest &.level-2 > .item {
-    @nest &:before {
-      content: '\25CB '; /* WHITE CIRCLE */
-    }
+  @nest &.level-2 > .item:before {
+    content: '  \25CB'; /* WHITE CIRCLE */
   }
 
-  @nest &.level-3 > .item {
-    @nest &:before {
-      content: '\25A0 '; /* BIG BLACK SQUARE */
-    }
+  @nest &.level-3 > .item:before {
+    content: '  \25A0'; /* BIG BLACK SQUARE */
   }
 
-  @nest &.level-5 > .item {
-    @nest &:before {
-      content: '\25CB '; /* WHITE CIRCLE */
-    }
+  @nest &.level-5 > .item:before {
+    content: '  \25CB'; /* WHITE CIRCLE */
   }
 
-  @nest &.level-6 > .item {
-    @nest &:before {
-      content: '\25A0 '; /* BIG BLACK SQUARE */
-    }
+  @nest &.level-6 > .item:before {
+    content: '  \25A0'; /* BIG BLACK SQUARE */
   }
 
-  @nest &.level-8 > .item {
-    @nest &:before {
-      content: '\25CB '; /* WHITE CIRCLE */
-    }
+  @nest &.level-8 > .item:before {
+    content: '  \25CB'; /* WHITE CIRCLE */
   }
 
-  @nest &.level-9 > .item {
-    @nest &:before {
-      content: '\25A0 '; /* BIG BLACK SQUARE */
-    }
+  @nest &.level-9 > .item:before {
+    content: '  \25A0'; /* BIG BLACK SQUARE */
   }
 }
 
 .bullet > .item {
   @nest &:before {
-    min-width: 1.5em; /* Make sure space between bullet and text never shrinks */
+    min-width: 1.5rem; /* Make sure space between bullet and text never shrinks */
```

Comment:
This should be relative to the text and not the root? (`em`)

Reply:
The font-size has been set to `7px`, so that's why it uses `rem` in this case.

Reply:
The font size is set to `7px`, so that's why `rem` is required in this case.

Reply:
If you set the font-size relative, then `em` makes sense. People can override the base font size in the portable text editor.